src/Microsoft.ML.FastTree/FastTreeTweedie.cs

```diff
@@ -37,7 +37,7 @@ public sealed partial class FastTreeTweedieTrainer : BoostingFastTreeTrainerBase
         public const string LoadNameValue = "FastTreeTweedieRegression";
         public const string UserNameValue = "FastTree (Boosted Trees) Tweedie Regression";
         public const string Summary = "Trains gradient boosted decision trees to fit target values using a Tweedie loss function. This learner is a generalization of Poisson, compound Poisson, and gamma regression.";
-        new public const string Remarks = @"<remarks>
+        public new const string Remarks = @"<remarks>
```

Comment:
Following the [documentation](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2010/435f1dw2(v=vs.100)), I see that keyword `new` is the first in statements.

Reply:
Let me add real @TomFinley 
RoboTom gives this:
warning TLC_ExplicitAccessModifiers: Symbol 'Remarks' did not have access modifiers as the leading modifiers

Reply:
The cold robotic hand of robotom was not programmed for this eventuality, due to the limitations of its flawed, human, designer. I might prefer to change robotom vs. do this.

Reply:
reverted. thank you for bringing this up Zeeshan!

---
In reply to: [199651709](https://github.com/dotnet/machinelearning/pull/478#discussion_r199651709) [](ancestors = 199651709)